vllm/v1/engine/__init__.py

```diff
@@ -161,6 +173,7 @@ class EngineCoreOutputs(
 
     utility_output: Optional[UtilityOutput] = None
     finished_requests: Optional[set[str]] = None
+    mfu_output: Optional[MFUOutput] = None
```

Comment:
I think it's probably fine to include this in `SchedulerStats`?

`SchedulerStats` is probably poorly named. AFAIR it's really "not-per-request stats" at this stage